lib/backend/k8s/resources/node_test.go

```diff
@@ -69,32 +69,14 @@ var _ = Describe("Test Node conversion", func() {
 		bgpIpv4Address := n.Value.(*apiv2.Node).Spec.BGP.IPv4Address
 		ipInIpAddr := n.Value.(*apiv2.Node).Spec.BGP.IPv4IPIPTunnelAddr
 		asn := n.Value.(*apiv2.Node).Spec.BGP.ASNumber
-		labels := n.Value.(*apiv2.Node).Labels
```

Comment:
What's the cause of this one?

Reply:
because it wasn't correct as per the current plan of storing the labels and annotations as encoded annotations.

This assumed the labels were passed straight through to the Calico node resource.  A bug.